### PR TITLE
[log] fix, log model view permissions

### PR DESCRIPTION
**Pull request by dpgaspar**
### CATEGORY

Choose one

- [X] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Restrict LogModelView permissions

### TEST PLAN
Tested that users don't have access to the View and Api through `LogRestApi`

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: tests/security_tests.py [hunk 1]
```diff
@@ -594,11 +594,16 @@ def test_is_admin_only(self):
             )
         )
 
-        self.assertTrue(
-            security_manager._is_admin_only(
-                security_manager.find_permission_view_menu("can_delete", "DatabaseView")
+        log_permissions = ["can_list", "can_show", "can_add", "can_edit", "can_delete"]
```
Comment (willbarrett) [inline tests/security_tests.py]: By default, I think we don't want logs to be editable. I'd remove `can_add`, `can_edit`, and `can_delete` for all users.

**Review by willbarrett — CHANGES_REQUESTED** [review]

File: tests/security_tests.py (hunk 1 above)
Comment (dpgaspar) [reply tests/security_tests.py]: Right! but that is being handled on #8960. I would merge #8960 first then on the solution found adapt this change to it, or merge this and then make the change on #8960

Comment (dpgaspar) [reply tests/security_tests.py]: it's merged!

Comment (willbarrett) [reply tests/security_tests.py]: OK, works for me.

**Review by willbarrett — APPROVED** [review]

**Pull request merged**